src/library/dao/autodjcratesdao.cpp

```diff
@@ -216,47 +216,47 @@ void AutoDJCratesDAO::createAndConnectAutoDjCratesDatabase() {
 
     // Be notified when a track is modified.
     // We only care when the number of times it's been played changes.
-    connect(m_pTrackCollection,
+    connect(m_pTrackCollectionManager->internalCollection(),
```

Comment:
Same here, should we forward these signals?

Reply:
Probably not, because the whole design is inappropriate. DAOs should be passive and not receive any signals.

TrackCollection with its interconnected DAOs is a maintenance hell full of side effects. I won't touch it again.